nextflow.config

```diff
@@ -95,7 +95,7 @@ params {
     gtdbtk_min_perc_aa                   = 10
     gtdbtk_min_af                        = 0.65
     gtdbtk_pplacer_cpus                  = 1
-    gtdbtk_pplacer_scratch               = true
+    gtdbtk_pplacer_scratchoff            = false
```

Comment:
It is a pretty minor thing, and maybe it is just me, but `gtdbtk_pplacer_scratchoff` sounds weird.  Would `gtdbtk_pplacer_use_stratch` be more intuitive?

Reply:
Ach yeah, I debated this for quite a while...

The _default_ was to use the scratch space method (as you can see with the `true`). This is makes GTDBTk more accessible/portable for people because it reduces the amount of memory required (as apparently pplacer can require a huge amount of memory). This is something appeals me to retain this as the default.

My issue with it was the original parameter was not very user friendly of having to manually specify `false` in `--gtdbtk_pplacer_scratch false`  to turn off this behaviour. By default to turn things on in Nextflow you only need to specify `--gtdbtk_pplacer_scratchoff`, which implicitly sets it to `true`, but I don't like this discordance in the way to set a boolean... so I changed the name of the parameter to ensure the behaviour is the same across the pipeline.

Reply:
Alternativelu could rename it to `--gtbdtk_pplacer_useram`  instead? Would that be more intuitive?

Reply:
I went with it, I think it would be more intuitive that way around